### PR TITLE
Code quality

### PR DESCRIPTION
**Pull request by dmitrytrager**
According to: https://redmine.smartdata.solutions/issues/4791

### CODE REVIEW

File: spec/controllers/api/v1/users/checkins_controller_privilege_spec.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@
   let(:developer) { FactoryGirl.create :developer }
   let(:user) { FactoryGirl.create :user }
   let(:device) { FactoryGirl.create :device, user_id: user.id, delayed: 10 }
-  let(:second_device) { FactoryGirl.create :device, user_id: user.id, delayed: 10 }
+  let(:second_device) { FactoryGirl.create :device, user_id: user.id, delayed: 10, name: "Second device" }
```
Comment (dmitrytrager) [inline spec/controllers/api/v1/users/checkins_controller_privilege_spec.rb]: I found that name collision is possible here

File: bin/quality [hunk 2]
```diff
@@ -2,7 +2,12 @@
 
 set -e
 
+# bin/rubocop --rails --format json --config config/rubocop.yml
```
Comment (dmitrytrager) [inline bin/quality]: commented until guys decide if they want to use these tools and the manner of using

File: config/rubocop.yml [hunk 3]
```diff
@@ -0,0 +1,56 @@
+require: rubocop-rspec
```
Comment (t-harps) [inline config/rubocop.yml]: We have a rubocop.yml file in the root of the project, shall we delete that in favour of this file?

Comment (dmitrytrager) [inline config/rubocop.yml]: @t-harps would you like some settings to be overwritten from your file?

Comment (t-harps) [inline config/rubocop.yml]: @dmitrytrager I think your file has all the relevant rules included, i will delete the old file.

Comment (dmitrytrager) [inline config/rubocop.yml]: ok, thanks

**Pull request merged**